stories/setup.stories.js

```diff
@@ -27,94 +27,130 @@ import { storiesOf } from '@storybook/react';
+			<WithTestRegistry registry={ registry } features={ [ 'storeErrorNotifications' ] }>
 				<SetupUsingGCP />
 			</WithTestRegistry>
 		);
+	}, {
+		decorators: [
+			withRegistry,
+		],
 	} );
 
 storiesOf( 'Setup / Using Proxy', module )
-	.add( 'Start', () => {
+	.add( 'Start', ( args, { registry } ) => {
+		provideSiteConnection( registry );
 		return (
-			<WithTestRegistry>
+			<WithTestRegistry registry={ registry }>
 				<SetupUsingProxy />
 			</WithTestRegistry>
 		);
+	}, {
+		decorators: [
+			withRegistry,
+		],
 	} )
-	.add( 'Start – with error', () => {
-		global._googlesitekitLegacyData.setup.isSiteKitConnected = false;
+	.add( 'Start – with error', ( args, { registry } ) => {
+		provideSiteConnection( registry, { connected: false } );
 		return (
-			<WithTestRegistry>
+			<WithTestRegistry registry={ registry }>
 				<SetupUsingProxy />
 			</WithTestRegistry>
 		);
+	}, {
+		decorators: [
+			withRegistry,
+		],
 	} )
-	.add( 'Start [User Input]', () => {
+	.add( 'Start [User Input]', ( args, { registry } ) => {
+		provideSiteConnection( registry );
 		return (
-			<WithTestRegistry features={ [ 'serviceSetupV2', 'userInput' ] }>
+			<WithTestRegistry
+				registry={ registry }
+				features={ [ 'serviceSetupV2', 'userInput' ] }
+			>
 				<SetupUsingProxy />
 			</WithTestRegistry>
 		);
+	}, {
+		decorators: [
+			withRegistry,
+		],
 	} )
-	.add( 'Start – with error [User Input]', () => {
-		global._googlesitekitLegacyData.setup.isSiteKitConnected = false;
-
+	.add( 'Start – with error [User Input]', ( args, { registry } ) => {
+		provideSiteConnection( registry, { connected: false } );
 		return (
-			<WithTestRegistry features={ [ 'serviceSetupV2', 'userInput' ] }>
+			<WithTestRegistry
+				registry={ registry }
+				features={ [ 'serviceSetupV2', 'userInput' ] }
+			>
 				<SetupUsingProxy />
 			</WithTestRegistry>
 		);
+	}, {
+		decorators: [
+			withRegistry,
+		],
 	} )
-	.add( 'Disconnected - URL Mismatch', () => {
-		const setupRegistry = ( registry ) => {
-			provideUserAuthentication( registry, {
-				authenticated: false,
-				disconnectedReason: DISCONNECTED_REASON_CONNECTED_URL_MISMATCH,
-			} );
-		};
+	.add( 'Disconnected - URL Mismatch', ( args, { registry } ) => {
+		provideSiteConnection( registry );
+		provideUserAuthentication( registry, {
+			authenticated: false,
+			disconnectedReason: DISCONNECTED_REASON_CONNECTED_URL_MISMATCH,
+		} );
 		return (
-			<WithTestRegistry callback={ setupRegistry }>
+			<WithTestRegistry registry={ registry }>
 				<SetupUsingProxy />
 			</WithTestRegistry>
 		);
+	}, {
+		decorators: [
+			withRegistry,
+		],
 	} )
-	.add( 'Disconnected - URL Mismatch [User Input]', () => {
-		const setupRegistry = ( registry ) => {
-			provideUserAuthentication( registry, {
-				authenticated: false,
-				disconnectedReason: DISCONNECTED_REASON_CONNECTED_URL_MISMATCH,
-			} );
-		};
+	.add( 'Disconnected - URL Mismatch [User Input]', ( args, { registry } ) => {
```

Comment:
These two look the same as the stories for "Start" and "Start [User Input]" – is that expected?

Reply:
Just pushed commits to address that :)